### ISSUE TITLE
Add information on account creation

### ISSUE THREAD

**Issue by nero** [theDavidCoen/BanksAgainstBitcoin#2]
Would it be possible to add another column with a link or information on how you can open accounts with these banks? Stuff like whether foreign nationals are allowed to have accounts and how you need to verify yourself (Id copy or like that).

**Comment by theDavidCoen** [comment]
Yes, but it would require a lot of research so I don't plan to add it right now.
Maybe in the future.

Thanks for the suggestion!
